lmdeploy/lite/apis/calibrate.py

```diff
@@ -132,6 +132,22 @@ class name or the class type itself.
             print(f'Move {mod_name} to GPU.')
 
 
+# TODO to be removed
+def make_compatible_internvl_config(model_path):
+    """Patch model.config since after transformers v4.45.0, InternVL models
+    can't use `save_pretrained`"""
+    if 'internvl' in model_path.lower():
```

Comment:
Let's use model's architecture instead of model_path to determine if it is an internvl model

Reply:
That need another config loading logic for each model.